sentry-spring-boot-jakarta/src/main/java/io/sentry/spring/boot/jakarta/SentryProperties.java

```diff
@@ -33,6 +33,8 @@ public class SentryProperties extends SentryOptions {
   /** Reactive framework (e.g. WebFlux) integration properties */
   private @NotNull Reactive reactive = new Reactive();
 
+  private boolean enableAotCompatibility = false;
```

Comment:
Can we have a comment here explaining the flag please. Basically what's written in the sample config above.